metron-interface/metron-config/cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "viewportWidth": 1435,
+  "viewportHeight": 850,
+  "video": false,
+  "pluginsFile": false,
+  "baseUrl": "http://localhost:4200"
+}
```

Comment:
You may want to add the default timeouts increases we have in the alerts ui to prevent the random sign in failure: https://github.com/apache/metron/pull/1323/files

Link to PR explaining why I increased these timeouts: https://github.com/apache/metron/pull/1323

Reply:
Good idea. Added.